native/src/main/java/io/ballerina/stdlib/persist/sql/datastore/SQLProcessor.java

```diff
@@ -56,9 +56,9 @@
  *
  * @since 0.3.0
  */
-public class MySQLProcessor {
+public class SQLProcessor {
```

Comment:
Shall we keep MySQLProcessor as well extending the SQLProcessor? otherwise it break the existing code.

Reply:
Done in [1fecb93](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/14/commits/1fecb931d87ca19f7efbef2f808662c28e17db4c)

Reply:
Done in [d653e9b](https://github.com/ballerina-platform/module-ballerinax-persist.sql/pull/14/commits/d653e9b9c20dc15153cc0dd37784e675e850b6e1)